src/main/proto/netflix/titus/agent.proto

```diff
@@ -321,6 +321,29 @@ message TaskIdentity {
     optional string ipv6Address = 5;
 }
 
+// For the identity client to query the metadata server to confirm the
+// currently running task's identity based on the whole pod
+message TaskPodIdentity {
+    // (Required) Current state of the task 
+    optional messages.TaskInfo taskInfo = 1;
+
+    // (Required) Pod details in bytes!
+    // > For the same reason, we do not recommend embedding these proto objects within your own proto definitions.
+    // > It is better to store Kubernetes objects as byte arrays, in the wire format, which is self-describing.
+    // > This permits you to use either JSON or binary (proto) wire formats without code changes.
+    // > It will be difficult for you to operate on both Custom Resources and built-in types otherwise.
+    optional bytes pod = 2;
+
+    // (Required) Timestamp, in Unix epoch seconds, when request was made.
+    optional uint64 unixTimestampSec = 3;
```

Comment:
Not sure if it's compatible with proto2, but could this be encoded as a `google.protobuf.timestamp.Timestamp`?

Reply:
We can. 

Should it be required?